pkg/services/file/file.go

```diff
@@ -86,9 +88,9 @@ type (
 	}
 )
 
-// NewFileService returns a new instance of the default implementation of FileService.
-// If nil is passed, the default FileService will be created with the default defaultFileServiceRuntime.
-func NewFileService(runtime defaultFileServiceRuntime) FileService {
+// NewFileService returns a new instance of the default implementation of Service.
+// If nil is passed, the default Service will be created with the default defaultFileServiceRuntime.
+func NewFileService(runtime defaultFileServiceRuntime) Service {
```

Comment:
We could also rename `defaultFileService` to `defaultService`, `defaultFileServiceRuntime` to `serviceRuntime` and `fileServiceDefaultRuntime` to `defaultServiceRuntime`. What you think?

Reply:
ok